tests/test_mc_rbdyn.cpp

```diff
@@ -64,6 +64,17 @@ BOOST_AUTO_TEST_CASE(TestRobotZMPSimple)
   const auto sensorNames = std::vector<std::string>{"LeftFootForceSensor", "RightFootForceSensor"};
   auto & lfs = robot.forceSensor("LeftFootForceSensor");
   auto & rfs = robot.forceSensor("RightFootForceSensor");
+  // Prevent using the JVRC1 calibration files (when they exist)
+  // for the ZMP computation to obtain repeatable results here.
+  // Resetting the calibrator makes sure that it has no effect.
+  //
+  // This test assumes that the sensor is at the model position and that there
+  // is no force offset.
+  //
+  // XXX We should investigate the effect of calibrator on ZMP measurement,
+  // and write a test that checks this case as well
+  lfs.resetCalibrator();
+  rfs.resetCalibrator();
```

Comment:
I suggest that instead of introducing `resetCalibrator()` we load a default calibrator for the tests

Reply:
This is what `resetCalibrator()` is doing (resetting all of the calibrator objects values to zero such that the calibrator has no effect on sensor measurements). Given that we have `ForceSensor::loadCalibrator` that allows loading a configuration file at any time, I don't really see anything wrong with having `ForceSensor::resetCalibrator` as well.

An alternative since this is only used for the unit test would be to call
```cpp
Eigen::Vector3d MC_RBDYN_DLLAPI zmp(const sva::ForceVecd & netTotalWrench,
                                    const sva::PTransformd & zmpFrame,
                                    double minimalNetNormalForce = 1.);
```
and compute the `netTotalWrench` from the `worldWrench` instead of `worldWrenchWithoutGravity`

instead of calling
```cpp
  Eigen::Vector3d Robot::zmp(const std::vector<std::string> & sensorNames,
                      const sva::PTransformd & zmpFrame,
                      double minimalNetNormalForce = 1.) const;
````

But that brings it a bit further from regular usage. I think the changes to the test are fine as-is, the point is only to run this test under its original assumptions, not to change anything. I will open a discussion regarding the use of the calibrator for such purposes, since I have the feeling that we are misusing it in that context.